pkg/constants/constants.go

```diff
@@ -19,7 +19,9 @@ const (
 	ElasticsearchPort = "9200"
 	FluentdName       = "fluentd"
 	KibanaName        = "kibana"
-	LogStoreService   = ElasticsearchFQDN + ":" + ElasticsearchPort
+	LogStoreURL       = "tls://" + ElasticsearchFQDN + ":" + ElasticsearchPort
+	// Secret used by cluster logging components (collector and store).
```

Comment:
This is not exactly correct.  The content of the secret may have the same entries but the ES output does not mount this secret

Reply:
Yes, you explained this to me before - I'll update the comment.